extensions/jitsi/common/jitsi.mjs

```diff
@@ -84,20 +117,20 @@ class JitsiHandler {
     console.log("onLocalTracks", tracks);
     this.localTracks = tracks;
     for (let i = 0; i < this.localTracks.length; i++) {
-      console.log(this.localTracks[i].getType());
-      if (this.localTracks[i].getType() === 'video') {
-        console.log('onLocalTracks insert video');
+      if (this.localTracks[i].getType() !== 'audio') {
```

Comment:
Optional: I would recommend an if statement for audio, another if statement for video cases and lastly an else statement to catch everything else. This way, we'll know if something unexpected shows up.